### DISCUSSION TITLE
How to keep a board refreshed periodically?

### DISCUSSION THREAD

**Discussion by mertcan79** [flet-dev/flet#253] [Q&A]
I am connecting to an SSE and I want the page to update itself and show the whole history.

**Comment by FeodorFitsner** [comment]
Flet does not use SSE. Could you please elaborate what you mean?

**Reply by mertcan79** [reply]
I have a FastAPI backend which Flet is taking data from through requests&sseclient modules. But it could be any datasource. I just want to refresh the page automatically.

**Comment by FeodorFitsner** [comment]
just make a loop in a separate thread with timeout inside where you update page's content and call `page.update()`?
